src/random_test.cpp

```diff
@@ -2610,9 +2610,10 @@ void alter_tablespace_rename(Thd1 *thd) {
 /* load special sql from a file */
 static std::vector<std::string> load_special_sql_from() {
   std::vector<std::string> array;
-  auto file = opt_string(SQL_FILE);
+  auto sql_file = opt_string(SQL_FILE);
```

Comment:
if user has give --sql-file=/path/to/test_grammer.sql This logic may fail.
as we need to use test_grammer.sql placed in /path/to/test_grammer.sql

Reply:
Fixed

Reply:
Note: The user options have been renamed
--grammar-file = <path to grammar file>
--grammer-sql = <option for probability of generating sqls using grammar file>